app/services/reject_application_by_default.rb

```diff
@@ -0,0 +1,14 @@
+class RejectApplicationByDefault
+  attr_accessor :application_choice
+
+  def initialize(application_choice:)
+    self.application_choice = application_choice
+  end
+
+  def call
+    #TODO: Do we need to include some other flag here to distinguish
```

Comment:
Good point, let's card this up and remove the comment.

Reply:
https://trello.com/c/VrqWtwch/1280-add-boolean-attribute-to-distinguish-application-choices-that-have-been-rejected-by-default